docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md

```diff
@@ -0,0 +1,39 @@
+# Existing EKS Cluster AWS Mixed Observability Accelerator
+
+## Objective
```

Comment:
Talk about the pattern and diagram will help. You can reuse mixed diagram

Reply:
done